.github/dependabot.yml

```diff
@@ -39,3 +39,10 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 1000
+  groups:
+    patch-updates:
+          applies-to: version-updates
+          update-types:
+          - "minor"
+          - "patch"
+
```

Comment:
If we are only wanting to group patches, wouldn't we just have "patch" here?

Reply:
oops, fixed @jpalac